### PR TITLE
DOCSP-41551: dropone() return type

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-41551>
Staging - https://preview-mongodbrustagir.gatsbyjs.io/golang/DOCSP-41551-dropone-return-type/fundamentals/indexes/#remove-an-index

[whats new
](https://preview-mongodbrustagir.gatsbyjs.io/golang/DOCSP-41551-dropone-return-type/whats-new/#what-s-new-in-2.0)

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

**Review by norareidy — APPROVED** [review]
lgtm!

File: source/fundamentals/indexes.txt [hunk 1]
```diff
@@ -440,31 +440,17 @@ The following example creates a unique, descending index on the ``theaterId`` fi
...
-      fmt.Println(res)
-
-   .. output::
-      :language: none
-      :visible: false
+.. code-block:: go
 
-      {"nIndexesWas": {"$numberInt":"2"}}
+   coll := client.Database("sample_mflix").Collection("movies")
+   err := coll.Indexes().DropOne(context.TODO(), "title_1")
```
Comment (prestonvasquez) [inline source/fundamentals/indexes.txt:453]: Why do we remove the error-handling portion of this example?

Comment (rustagir) [reply source/fundamentals/indexes.txt:453]: Oops, should have just removed the print statement, i will add the error handling back in!

Comment (rustagir) [reply source/fundamentals/indexes.txt:453]: fixed!

**Review by prestonvasquez — APPROVED** [review]

**Pull request merged**